pkg/operator/ceph/cluster/mon/spec.go

```diff
@@ -174,6 +175,13 @@ func (c *Cluster) makeMonDaemonContainer(monConfig *monConfig) v1.Container {
 		),
 		Resources: c.resources,
 	}
+
+	// If deploying Nautilus and newer we need a new port of the monitor container
+	if cephv1.VersionAtLeast(c.cephVersion.Name, cephv1.Nautilus) {
+		addContainerPort(container, "msgr2", 3300)
+	}
+
+	return container
 }
 
 func joinHostPort(host string, port int32) string {
```

Comment:
I think this can be deleted since the usage of `joinHostPort` is removed in lines 158 and 159.

Reply:
That's what I thought as well, but still I thought it'll be useful to keep it around maybe for later?

Reply:
The method is pretty simple. I'd rather not have code that doesn't have use and have to implement this later than to keep it and find we may never need it.